_posts/blog/2021-01-18-Open-position-Cryptography-Engineer.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Open Position: Cryptography Engineer"
+excerpt: "You can find application details below."
+categories: blog
+tags: [staff]
+date: 2021-01-18
+author: antonie
+---
+
+
+The Zcash Foundation is a 501(c)(3) nonprofit, public charity that builds and supports privacy infrastructure for the public good. We work on strengthening financial privacy with technology, focused on the Zcash protocol and blockchain.
+
+We're looking for someone who is as excited as we are about building private financial infrastructure for the public good, and we take that task very seriously.
+
+# Why work for the Zcash Foundation?
+* We are a small start-up nonprofit that punches way above its weight.
+* Everything we build is open source, and we develop in the open.
+* Our mission and values are very important. We want to work with like-minded individuals who share them with us.
+* We are fully remote and have team members all over the globe.
+* Leadership cares deeply about your health and fulfillment.
+* We offer comprehensive benefits that include full coverage health insurance for employees and dependents, retirement savings account, parental leave, Bring your own Device (BYOD) annual benefit, and a generous Paid Time Off policy.
+* Nonprofit means what it says! We build what serves our mission.
+# Role: Cryptography Engineer
```

Comment:
The formatter needs a newline, otherwise it will indent the heading

```suggestion
* Nonprofit means what it says! We build what serves our mission.

# Role: Cryptography Engineer
```